runtime/server/controller.go

```diff
@@ -176,7 +176,7 @@ func (s *Server) WatchResources(req *runtimev1.WatchResourcesRequest, ss runtime
 			var err error
 			r, access, err = s.applySecurityPolicy(ss.Context(), req.InstanceId, r)
 			if err != nil {
-				s.logger.Info("failed to apply security policy", zap.String("name", n.Name), zap.Error(err))
+				s.logger.Debug("failed to apply security policy", zap.String("name", n.Name), zap.Error(err))
```

Comment:
should we make it `error` ?

Reply:
Thought about it, but I think it will trigger if a user has broken template logic in their security policy. And we don't really want error-level logs for user errors.